nixos/tests/systemd-initrd-simple.nix

```diff
@@ -2,16 +2,19 @@ import ./make-test-python.nix ({ lib, pkgs, ... }: {
   name = "systemd-initrd-simple";
 
   nodes.machine = { pkgs, ... }: {
-    boot.initrd.systemd = {
-      enable = true;
-      emergencyAccess = true;
```

Comment:
emergencyAccess got disabled from here intentionally? Or we just didn't use it?

Reply:
@flokli We didn't use it. I think it was just something for testing purposes that made it into the initial sd-initrd PR back in 2022 by mistake, and it's been bothering me ever since :P